docs/source/openvino/optimization.mdx

```diff
@@ -149,7 +149,7 @@ Click on a ✅ to copy the command/code for the corresponding optimization case.
             <td style="text-align: center; vertical-align: middle;">–</td>
         </tr>
         <tr>
-            <td style="text-align: center; vertical-align: middle;">text-to-image<br>(OVStableDiffusionPipeline)</td>
+            <td style="text-align: center; vertical-align: middle;">text-to-image, text-to-video<br>(OVDiffusionPipeline)</td>
```

Comment:
I think we would also need to update :

https://github.com/huggingface/optimum-intel/blob/f2a98c7b9accd6aa7c54dd12bc775a803306b198/docs/source/openvino/optimization.mdx?plain=1#L161

https://github.com/huggingface/optimum-intel/blob/f2a98c7b9accd6aa7c54dd12bc775a803306b198/docs/source/openvino/optimization.mdx?plain=1#L175

https://github.com/huggingface/optimum-intel/blob/f2a98c7b9accd6aa7c54dd12bc775a803306b198/docs/source/openvino/optimization.mdx?plain=1#L187

to replace `OVStableDiffusionPipeline` with `OVDiffusionPipeline`